requirements.txt

```diff
@@ -12,7 +12,7 @@ Jinja2==2.10.3
 marshmallow-enum==1.5.1
 marshmallow-sqlalchemy==0.19.0
 marshmallow==3.2.1
-psycopg2==2.8.4
+psycopg2-binary==2.8.4
```

Comment:
I wanted to leave this change in for a discussion: Do we need to compile psycopg2 when installing dependencies? If not, we should start using `psycopg2-binary`

Reply:
Sounds good! I did not know psycopg2-binary was not recommended for prod environment. I'll remove the changes on this requirements.txt 👍

Reply:
It's mentioned in its [readme on pypi](https://pypi.org/project/psycopg2-binary/) ;)

> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

<details>
FWIW, they may be a bit overly cautious - in older psycopg2 versions binary wheels were the default so many people ended up using those implicitly in the normal psycopg2 package.
</details>